### PR TITLE
dotCMS/core#26044 add icons to experiment status

### LINKED ISSUE #26044: [UI] Add icons to Experiment' status
### Parent Issue

_No response_

### User Story

Add icons to Experiment' status

### Acceptance Criteria

Add icons to Experiment' status

### Proposed Objective

User Experience

### Proposed Priority

Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

https://www.figma.com/file/7CzbMcGSbTIJjerWtcapAm/Design-System-2023?type=design&node-id=1312%3A22690&mode=design&t=HBxruAJsaFZUhB0d-1

Screenshot 2023-09-11 at 11 54 01 AM

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by oidacra**
### Proposed Changes
* Add icons to experiment status
* Replace `p-tag` with `p-chip`

### Checklist
- [x] Tests

### Screenshots
Screenshot 2023-09-19 at 12 40 34 PM
Screenshot 2023-09-19 at 12 40 43 PM
Screenshot 2023-09-19 at 12 40 51 PM
Screenshot 2023-09-19 at 12 41 00 PM

### <samp>🤖 Generated by Copilot at c769d81</samp>

This pull request improves the UI and accessibility of the dotCMS theme and components, by using the PrimeNG Chip component, adjusting the color variables, simplifying the SCSS code, and adding a Storybook story for the Chip component. It affects the files `core-web/libs/dotcms-scss/*`, `core-web/libs/portlets/dot-experiments/*`, `core-web/apps/dotcms-ui/.storybook/preview.js`, `core-web/apps/dotcms-ui/project.json`, `core-web/apps/dotcms-ui/src/stories/primeng/chip/Chip.stories.ts`, and `core-web/libs/dotcms-models/src/lib/dot-experiments-constants.ts`.

## Related Issue
Fixes #26044 

# Explanation of Changes

### <samp>🤖 Generated by Copilot at c769d81</samp>

*  Add a new Storybook story for the PrimeNG Chip component ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-44af4dfb0381252eacd42fb9a914b7e8690212f6fdf3df528bc73cc8a341a4a5R1-R80))
*  Update the styles for the p-chip component, to use the accessible color variables and to match the design system specifications ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-864f8377caaa673a5f527e429f3c8d8de0584eaca07bb5dad3eebb3cbd2f91a1L2-R65))
*  Replace the p-tag component with the p-chip component in the `dot-experiments-ui-header.component.html` template, to display the experiment status ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-f2a73e04960c8fef0a62157e1d67cb80f66453e5a11c70bbce4aea1714ef0f79L15-R17), [link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-f2a73e04960c8fef0a62157e1d67cb80f66453e5a11c70bbce4aea1714ef0f79L26-R27))
*  Add the OnChanges interface, the ChipModule import, the statusIcon property, and the ngOnChanges method to the `dot-experiments-ui-header.component.ts` class, to use the p-chip component and to update the icon based on the experiment status input ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-802911a57efe89948c36c7122bdb9eb088b33ee971c5d95a8e952881a9ac7681L2-R14), [link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-802911a57efe89948c36c7122bdb9eb088b33ee971c5d95a8e952881a9ac7681R21), [link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-802911a57efe89948c36c7122bdb9eb088b33ee971c5d95a8e952881a9ac7681L33-R48), [link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-802911a57efe89948c36c7122bdb9eb088b33ee971c5d95a8e952881a9ac7681L50-R71))
*  Add the ExperimentsStatusIcons constant object to the `dot-experiments-constants.ts` file, to map the experiment status to the icon class name ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-f1a1a6a6f21372c106dd50983b66d511523dd176656f6d368fe830a87a26b09cR43-R50))
*  Add some global styles for the .pi class, to handle the size and alignment of the icons, and to use variables for the font-size and width values ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-c11f0af3dcde8c3bb08b1cfa6214b4c17a9172f0baf3bb254554e6c89809d19aR58-R78))
*  Add some variables for the icon box sizes, to be used in the global styles for the .pi class, and to match the design system specifications ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-560fcfc492a8e2e54af1146b4ad7c9336d90c60ef64c673e8b5be2f2163de929L15-R21))
*  Reorder the stylesheets imported in the `project.json` file, to fix a CSS issue with the PrimeIcons font ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-a5a7709ae811f21c30177ca89ea61196bd970f2695ce9a24aee461b0e792dfa8L38-R39))
*  Add a parameter to the Storybook configuration, to expand the controls panel by default ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-6dc455a2ae6ec4a9cf88446ff04094bd08a301131a39fa38413c25b1bd2e7f0aR2))
*  Update the color variables for the accessible text blue and blue background, to improve the contrast ratio and accessibility ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-a673541496e37f0c8d73eef81a2026235e3d621daf64af8bf35a97881cb2dce5L107-R106))
*  Update the color variable for the anchor element in the `dot-experiments-ui-header.component.scss` styles, to use the accessible color variable ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-3fe2e334124588ab84e7834a7b68db4523773a3de300fef6a3a95e334fea724dL13-R13))
*  Update the color variables for the p-tag-info class, to use the accessible color variables ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-852b51b6c0cca889f1ba93e8834dd9ae9157ec88fbc30a91eb14e4a0254b7632L40-R41))
*  Remove redundant or unnecessary font-size declarations for the .pi and .p-button-icon classes, to avoid conflicts and duplication ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-c11f0af3dcde8c3bb08b1cfa6214b4c17a9172f0baf3bb254554e6c89809d19aL24-R27), [link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-9cc81ddc4d2c27cb7a9e0c503c2b592abb371ed8dce7f55b34f0c9dd53aa46f4L20), [link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-9cc81ddc4d2c27cb7a9e0c503c2b592abb371ed8dce7f55b34f0c9dd53aa46f4L101-R102), [link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-291473233533222487f2c2d6729e1c47bf788e43eeed2aab132f3123f71ce002L29-L33))
*  Remove unused color variables for the alert blue and light blue colors, since they are replaced by the accessible color variables ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-a673541496e37f0c8d73eef81a2026235e3d621daf64af8bf35a97881cb2dce5L87-L88))
*  Remove an empty line in the `_misc.scss` file ([link](https://github.com/dotCMS/core/pull/26177/files?diff=unified&w=0#diff-c11f0af3dcde8c3bb08b1cfa6214b4c17a9172f0baf3bb254554e6c89809d19aR37))

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/stories/primeng/chip/Chip.stories.ts
Comment (zJaaal) [inline core-web/apps/dotcms-ui/src/stories/primeng/chip/Chip.stories.ts:1]: do we need this changes?

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-header/dot-experiments-ui-header.component.spec.ts [hunk 1]
```diff
@@ -2,8 +2,9 @@ import { byTestId, createComponentFactory, Spectator } from '@ngneat/spectator/j
 
 import { DatePipe } from '@angular/common';
 
+import { Chip } from 'primeng/chip';
 import { Skeleton } from 'primeng/skeleton';
-import { Tag, TagModule } from 'primeng/tag';
+import { TagModule } from 'primeng/tag';
```
Comment (zJaaal) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-header/dot-experiments-ui-header.component.spec.ts:7]: do we need this module?

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-header/dot-experiments-ui-header.component.ts [hunk 2]
```diff
@@ -30,11 +40,12 @@ import { DotPipesModule } from '@pipes/dot-pipes.module';
         DotMessagePipe,
         SkeletonModule,
         TagModule,
```
Comment (zJaaal) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-header/dot-experiments-ui-header.component.ts:42]: do we need this module?

**Review by zJaaal — CHANGES_REQUESTED** [review]

**Review by zJaaal — APPROVED** [review]

**Review by rjvelazco — APPROVED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-header/dot-experiments-ui-header.component.ts (hunk 2 above)
Comment (oidacra) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-header/dot-experiments-ui-header.component.ts:42]: No more. Removing.

**Review by hmoreras — APPROVED** [review]

**Pull request merged**